ua-snap/iem-webapp
# Only valid clicks on Alaska sent to API

This PR adds a GeoJSON of Alaska over the map that prevents anywhere outside of that area from sending a click event to the function responsible for contacting the API, and modifies the UX to make the interactive area more clear.

To test this:
 - Load the app, scroll down to the map.  It should show a "mask" layer (work in progress but OK to merge this because we can sort out the background layer separately).
 - Roll the mouse inside/outside the valid/invalid areas, observe that it's a clickable pointer when you're inside and not a clicker when outside
 - Click outside the valid data mask.  It shouldn't do anything.
 - Click inside the valid data mask.  It should load the point as usual.  (Reload or go back to the map).
 - Go to the Lat/Lng picker.  It should have no error message now, but the Go button is disabled.
 - Enter nonsense characters, i.e. "a02k3mawel".  It should say it can't figure out what to do & "Go" button is disabled.
 - Clear out the invalid entry.  It should clear the error message when there's <8 chars (this is so the user doesn't instantly get an error message as they type).
 - Enter an invalid point:  63.95, -168.00.  It should say that the point is outside the domain of the datasets, and the Go button is disabled.
 - Clear out the invalid entry.
 - Enter a valid point: 70.87°N, 157.73°W.  It should clear the error message and the Go button should be clickable.

Fixes #49
